CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 - Set `Content-Security-Policy: frame-ancestors 'self'; default-src 'self'`
   HTTP header for HTTP 404 responses as well ([rhbz#2160664])
 - Validate dates in location constraint rules ([ghpull#644])
+- Fix displaying differences between configuration checkpoints in
+  `pcs config checkpoint diff` command ([rhbz#2175881])
```

Comment:
Shouldn't this be in the unreleased section?

Reply:
Yes, it should be. I moved it to the unreleased section.